src/components/Accordion/Accordion.spec.tsx

```diff
@@ -92,7 +92,6 @@ describe('Components / Accordion', () => {
       render(
         <>
           <TestAccordion />
-          {/* eslint-disable-next-line jsx-a11y/role-has-required-aria-props */}
```

Comment:
Have you run `yarn lint` to confirm this works? The comment is there for a reason - `<button>`s are only supposed to have `role="button"`. This test should just be rewritten to make this ESLint warning not come up in the first place but, otherwise, you'll need to restore this line.